examples/rust_lang_tester/lang_tests/no_main.rs

```diff
@@ -0,0 +1,11 @@
+// [Compiler]
+// status = "error"
+// stderr = """
```

Comment:
The triple-quoting required is a minor annoyance, but it's not really enough to sufficiently upset me though.

Reply:
I suppose an alternative approach, which is used in Rust's `compiletest` framework, is to have separate files for the resulting program's `stdout` and `stderr`. For example, `no_main.rs` would have a corresponding `no_main.rs.stdout` etc.. and these files are then read and compared to the actual output.

The benefit of this is we need less (or maybe even none at all if we're clever) language specific pattern matching code from the user. The problem is I can't think of a clean way to separate these on a per-command basis. If we're testing both compilation _and_ runtime, we'd end up with around 5 files per single test which might be overkill. Rust solves this by splitting tests into separate `compile_*` and `run_*` directories, but at this point I wonder if we'd just end up reinventing the wheel.

Reply:
I already mentioned this approach to Laurie, and he's not up for it ;)

Reply:
Yeah, I definitely don't want to do this, because the API would become much more complex.